libs/dal/src/repositories/subscriber/subscriber.repository.ts

```diff
@@ -32,6 +33,68 @@ export class SubscriberRepository extends BaseRepository<SubscriberDBModel, Subs
     );
   }
 
+  async bulkCreateSubscribers(subscribers, environmentId: EnvironmentId, organizationId: OrganizationId) {
```

Comment:
Wondering what type I should use for subscribers, as they are actually of the dto type defined in the api / the ISubscriberDefine in shared

Reply:
I think we should use the same types. This is one of the parts that I wanted to reuse from the trigger event subscriber on-the-fly creation to have one single source of truth.